### ISSUE TITLE
[Suggestion] Add chrystals/ors to loot-table of mines

### ISSUE THREAD

**Issue by LoneDev6** [LoneDev6/ItemsAdder#98]
### Add chrystals/ors to loot-table of mines
If possible would it be really cool to add the chrystals/ores to the loot table used for mineshafts, so that they would appear in chest minecarts.

**Comment by LoneDev6** [comment]
That's a really cool idea, I have to figure out if there's a "clean" way to do that without hacky tricks

**Comment by LoneDev6** [comment]
Interesting, seems it got added on 1.15.2, I missed this addition.
I will add this in the next update.

**Comment by LoneDev6** [comment]
This is not doable since there is no event for loot chests, only for minecarts.
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/browse/src/main/java/org/bukkit/craftbukkit/CraftLootTable.java?at=de42aa13f387632b360c5b1b003b212499f3f4f2#63

```java
// TODO: When events are added, call event here w/ custom reason?
```

I'll consider implementing this feature as soon as Spigot provides an API event for that.

**Comment by Akiranya** [comment]
Hi Lone, this event will be called from within the `net.minecraft.world.level.storage.loot.LootTable#fillInventory` for loot chests, and this event is already used by [EcoEnchants](https://github.com/Auxilor/EcoEnchants/blob/master/eco-core/core-plugin/src/main/kotlin/com/willfp/ecoenchants/mechanics/LootSupport.kt#L22) to add enchantment books to loot chest in the spigot platform.

Not sure how that "TODO" was left there... and I think we don't acutally need to use the CraftLootTable to modify the loot chest content (as whta has been done by EcoEnchants)

**Comment by LoneDev6** [comment]
So basically this TODO text is outdated and the event is triggered also for normal chests?

**Comment by Akiranya** [comment]
yep the TOOD text is outdated. I already implemented this feature on my own custom enchantment plugin and it works like a charm
